include/mbedtls/ssl.h

```diff
@@ -2914,6 +2913,19 @@ void mbedtls_ssl_conf_dhm_min_bitlen( mbedtls_ssl_config *conf,
  * \note           This list should be ordered by decreasing preference
  *                 (preferred curve first).
  *
+ * \note           The default list is the same set of curves that
+ *                 #mbedtls_x509_crt_profile_default allows, plus
+ *                 ECDHE-only curves selected according to the same criteria.
```

Comment:
Note that the X.509 and TLS profiles are currently aligned, but we don't promise that they will remain so forever.